### ISSUE TITLE
Config becomes corrupt when computer crashes

### ISSUE THREAD

**Issue by steel9** [t1m0thyj/WinDynamicDesktop#144]
When WinDynamicDesktop is running, and the computer crashes, due to a power outage for instance, the config file becomes corrupt (filled with NUL signs), which throws an error on application start:
```
[2019-07-23 10:00:52.250] System.NullReferenceException: Object reference not set to an instance of an object.
   at WinDynamicDesktop.JsonConfig.LoadConfig()
   at WinDynamicDesktop.AppContext..ctor(String[] args)
   at WinDynamicDesktop.Program.Main(String[] args)
```
Windows version: Windows 10 1903
WinDynamicDesktop version: 3.3.0 UWP

Maybe perform some null checks after loading the config to make sure it's not corrupted, and store a backup config that can be restored if this happens (that is updated when the normal config file is updated).

**Comment by FantasiaChan** [comment]
[Google Translate]
You can reinstall the application instead.

**Comment by t1m0thyj** [comment]
Fixed by #161, thanks @Senexis
